File: easy/challenge-270/python/main.py
import sys


def main():
    result = []
    for i, line in enumerate(sys.stdin):
        for j, ch in enumerate(line[:-1]):
            while len(result) < (j + 1):
                result.append([])
            while len(result[j]) < (i + 1):
                result[j].append(' ')

            result[j][i] = ch

    for line in result:
        for ch in line:
            sys.stdout.write(ch)
        sys.stdout.write('\n')

if __name__ == '__main__':
    main()

File: easy/challenge-270/nim/main.nim
#!env nim-run
import streams

var i = 0
var result : seq[seq[char]] = newSeq[seq[char]]()

while not endoffile(stdin):
    let line = stdin.readLine
    for j, ch in line[0..<line.len]:
        while result.len < (j + 1):
            result &= newSeq[char]()

        while result[j].len < (i + 1):
            result[j] &= ' '

        result[j][i] = ch
    i += 1

for line in result:
    for ch in line:
        stdout.write(ch)
    stdout.write("\n")
